src/entities/twinFlowTransition/libs/hooks/useFilters.ts

```diff
@@ -0,0 +1,109 @@
+import { AutoFormValueInfo, AutoFormValueType } from "@/components/auto-field";
+import { useTwinStatusSelectAdapter } from "@/entities/twinStatus";
+import {
+  toArray,
+  toArrayOfString,
+  wrapWithPercent,
+  type FilterFeature,
+} from "@/shared/libs";
+import {
+  TF_Transition,
+  TF_Transition_DETAILED,
+  TF_TransitionFilterKeys,
+  TF_TransitionFilters,
+  useTwinFlowTransitionSearchV1,
+} from "../../api";
+
+type FilterKeys = Exclude<TF_TransitionFilterKeys, "twinflowIdList">;
+
+export function useTF_TransitionFilters(
+  twinClassId?: string
+): FilterFeature<FilterKeys, TF_TransitionFilters> {
+  const { searchTwinFlowTransitions } = useTwinFlowTransitionSearchV1();
+  const { getById, getItems, getItemKey, getItemLabel } =
+    useTwinStatusSelectAdapter(twinClassId);
+
+  function buildFilterFields(): Record<FilterKeys, AutoFormValueInfo> {
+    return {
+      aliasLikeList: {
+        type: AutoFormValueType.combobox,
+        label: "Alias",
+        multi: true,
+        getById: async (key: string) =>
+          [].find((i: TF_Transition) => i.alias === key),
```

Comment:
Should at least add a TODO to replace this logic later instead of looking for alias in an empty array